html2docx/html2docx.py

```diff
@@ -78,6 +78,8 @@ def _reset(self) -> None:
 
         # Formatting options
         self.pre = False
+        self.table_cell: Optional[Any] = None
+        self.tables: List[Tuple[Any, int, int]] = []
```

Comment:
Since the table is only appended to, we could only keep track of the column and always use the last row. NBD either way.